src/components/InsureeSummary.js

```diff
@@ -19,10 +21,15 @@ const useStyles = makeStyles(() => ({
   label: {
     textAlign: "right",
   },
+  button: {width: 130, height: 40, background: "#F5F5F5"}
```

Comment:
Instead of styles try to use "variant" and "color". This will ensure that the button looks like other buttons in IMIS.